File: 백준/Bronze/3449. 해밍 거리/해밍 거리.cc
#include <iostream>
using namespace std;
int main(void) {
    int t, cnt;
    string x, y;
    cin >> t;
    for (int i = 0; i < t; i++) {
        cin >> x;
        cin >> y;
        cnt = 0;
        for (int j = 0; j < x.size(); j++) {
            if (x[j] != y[j])
                cnt += 1;
        }
        cout << "Hamming distance is " << cnt << ".\n";
    }
    return 0;
}

File: 백준/Bronze/3449. 해밍 거리/해밍 거리.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    x,y=input(),input()
    cnt=0
    for i in range(len(x)):
        if x[i]!=y[i]:
            cnt+=1
    print("Hamming distance is %d." % cnt)